<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskNew</class>
 <widget class="QDialog" name="taskNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame{border-image: url(:/background/tasknew.png);
border-top-left-radius:15px;border-top-right-radius:15px;border-bottom-left-radius:15px;border-bottom-right-radius:15px;}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1921</width>
     <height>1081</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>翻译性质：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>中译英</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英译中</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>299</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>翻译标题：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="title">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入翻译标题</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>165</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>任务简介：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="introEdit">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>170</y>
      <width>491</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>请输入任务简介</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>440</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>翻译任务：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="translateTask">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>450</y>
      <width>491</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>请输入翻译任务</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>435</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>任务周期：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>456</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>天</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>580</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>负责人报名截止日期：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leaderYear">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>590</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>600</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>年</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leaderMonth">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>590</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>599</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leaderDay">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>590</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>600</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>735</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>任务总金额：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="money">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>730</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>745</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>元</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confrmBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>870</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="canclBtn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>870</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Main">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>1000</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>返回主界面</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
